storage/imxrt-flash/flashsrv.c

```diff
@@ -79,10 +81,10 @@ typedef struct {
 struct {
 	flash_memory_t flash_memories[FLASH_MEMORIES_NO];
 	uint32_t flexspi_addresses[FLASH_MEMORIES_NO];
+	flashsrv_partitionCtx_t ctx;
```

Comment:
nitpick: maybe `ops` better suits (operations) function vector, instead of `ctx`, also  in type name `flashsrv_partitionOps_t`

Reply:
Good idea, thx